sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -252,6 +252,21 @@ tablePropertyKey
     | STRING
     ;
 
+optionParameterList
+    : '(' optionParameter (',' optionParameter)* ')'
+    ;
+
+optionParameter
+    : key=tablePropertyKey (EQ? value=optionValue)?
```

Comment:
We could remove `EQ?` here. This is actually not supported by data source tables.